backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.adoc

```diff
@@ -0,0 +1,29 @@
+:_mod-docs-content-type: ASSEMBLY
+
+[id="oadp-1-4-release-notes"]
+= OADP 1.4 release notes
+
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: oadp-1-4-release-notes
+
+toc::[]
+
+The release notes for OpenShift API for Data Protection (OADP) describe new features and enhancements, deprecated features, product recommendations, known issues, and resolved issues.
+
+include::modules/oadp-1-4-0-release-notes.adoc[leveloffset=+1]
+include::modules/oadp-backing-up-dpa-configuration-1-4-0.adoc[leveloffset=+3]
+include::modules/oadp-upgrading-oadp-operator-1-4-0.adoc[leveloffset=+3]
+[role="_additional-resources"]
+.Additional resources
+* xref:../../../operators/admin/olm-upgrading-operators.adoc#olm-changing-update-channel_olm-upgrading-operators[Updating installed Operators]
+
+[id="oadp-converting-dpa-to-new-version-1-4-0_{context}"]
+=== Converting DPA to the new version
+
+To upgrade from OADP 1.3 to 1.4, no Data Protection Application (DPA) changes are required
```

Comment:
Missing a period at the end of this sentence.

Reply:
fixed